spec/features/deleting_files_spec.rb

```diff
@@ -9,6 +9,5 @@
     instance.store!(image)
   end
 
-  it 'deletes the image when assigned a `nil` value' do
-  end
+  it 'deletes the image when assigned a `nil` value'
```

Comment:
No point in retaining this pending spec. It hasn't been addressed in years, and I don't plan on fighting with the eval/callback system in CarrierWave to make it happen.